sympy/solvers/tests/test_ode.py

```diff
@@ -3768,3 +3768,30 @@ def test_2nd_2F1_hypergeometric():
           C2*hyper((S(1)/21, -S(11)/14), (S(5)/7,), x**(S(2)/7)))/(x**(S(2)/7) - 1)**(S(19)/84))
     assert sol == dsolve(eq, hint='2nd_hypergeometric')
     # assert checkodesol(eq, sol) == (True, 0) #issue-https://github.com/sympy/sympy/issues/17702
+
+def test_issue_12623():
+	t = symbols("t")	
+	u = symbols("u",cls=Function)	
```

Comment:
We should use a 4 space indent in sympy.

Also I think it would be better if these tests were together with the other tests for undetermined coefficients.

Reply:
https://github.com/sympy/sympy/blob/7662acf46b7294d4ef16f7efd27052940bd06180/sympy/solvers/tests/test_ode.py#L2072

Should i add them here?

Reply:
Yes. Perhaps not in the same function but nearby (e.g. just below).

Reply:
Yes. Perhaps not in the same function but nearby (e.g. just below).